src/main/java/org/codehaus/staxmate/dom/Tupel.java

```diff
@@ -0,0 +1,160 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+
+package org.codehaus.staxmate.dom;
+
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ *
+ * @author elbosso
+ */
+public class Tupel<T,U> extends java.lang.Object
```

Comment:
Misspelling, should he `Tuple`. Also, no need to indicate `extends Object`, that's implicit.

Reply:
Also: for purposes here, looks bit like overkill; maybe trim all methods not used.

Reply:
sorry for the typo - shows i'm a German...
I left it in for the time being